doc/source/dev/roadmap-detailed.rst

```diff
@@ -340,8 +347,9 @@ isolve:
 dsolve:
 
 - add sparse Cholesky or incomplete Cholesky
-- look at CHOLMOD
-
+- add sparse QR
+- improve interface to SuiteSparse UMFPACK
+- add interfaces to SuiteSparse CHOLMOD and SPQR
 
 Ideas for new features:
```

Comment:
@mckib2 and I will be finishing the PROPACK wrapper (suggested in the first invisible line) for a NumFOCUS SDG.